source/fundamentals/indexes.txt

```diff
@@ -68,8 +68,8 @@ MongoDB supports a number of different index types to support querying
 your data. The following sections describe the most common index types
 and provide sample code for creating each index type.
 
-Single Field and Compound Indexes
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Single Field
```

Comment:
Issue: Title isn't specific enough for the reader to identify it as an index
Suggestion: Single Field -> Single Field Indexes